packages/docs/src/data/pages/directives/Intersect.json

```diff
@@ -21,6 +21,15 @@
         }
       ]
     },
+    {
+      "type": "section",
+      "children": [
+        {
+          "type": "examples",
+          "value": ["simple/once"]
```

Comment:
would suggest renaming the example to `options` to stay consistent with the example header.